filament/src/PostProcessManager.cpp

```diff
@@ -723,41 +724,48 @@ FrameGraphId<FrameGraphTexture> PostProcessManager::screenSpaceAmbientOcclusion(
+                "Duplicate Depth Pass",
+                [&](FrameGraph::Builder& builder, auto& data) {
+                    data.input = builder.read(depth, FrameGraphTexture::Usage::BLIT_SRC);
 
-                auto desc = builder.getDescriptor(data.input);
-                desc.levels = 1; // only copy the base level
+                    auto desc = builder.getDescriptor(data.input);
+                    desc.levels = 1;// only copy the base level
 
-                // create a new buffer for the copy
-                data.output = builder.createTexture("Depth Texture Copy", desc);
+                    // create a new buffer for the copy
+                    data.output = builder.createTexture("Depth Texture Copy", desc);
 
-                // output is an attachment
-                data.output = builder.write(data.output,
-                        FrameGraphTexture::Usage::BLIT_DST);
-            },
-            [=](FrameGraphResources const& resources, auto const& data, DriverApi& driver) {
-                auto const& src = resources.getTexture(data.input);
-                auto const& dst = resources.getTexture(data.output);
-                auto const& srcSubDesc = resources.getSubResourceDescriptor(data.input);
-                auto const& dstSubDesc = resources.getSubResourceDescriptor(data.output);
-                auto const& desc = resources.getDescriptor(data.output);
-                assert_invariant(desc.samples == resources.getDescriptor(data.input).samples);
-                // here we can guarantee that src and dst format and size match, by construction.
-                driver.blit(
-                        dst, dstSubDesc.level, dstSubDesc.layer, { 0, 0 },
-                        src, srcSubDesc.level, srcSubDesc.layer, { 0, 0 },
-                        { desc.width, desc.height });
-            });
+                    // output is an attachment
+                    data.output = builder.write(data.output, FrameGraphTexture::Usage::BLIT_DST);
+                },
+                [=](FrameGraphResources const& resources, auto const& data, DriverApi& driver) {
+                    auto const& src = resources.getTexture(data.input);
+                    auto const& dst = resources.getTexture(data.output);
+                    auto const& srcSubDesc = resources.getSubResourceDescriptor(data.input);
+                    auto const& dstSubDesc = resources.getSubResourceDescriptor(data.output);
+                    auto const& desc = resources.getDescriptor(data.output);
+                    assert_invariant(desc.samples == resources.getDescriptor(data.input).samples);
+                    // here we can guarantee that src and dst format and size match, by
+                    // construction.
+                    driver.blit(dst, dstSubDesc.level, dstSubDesc.layer, {0, 0}, src,
+                            srcSubDesc.level, srcSubDesc.layer, {0, 0}, {desc.width, desc.height});
```

Comment:
could we keep the original formatting here? it makes it easier to read the src/dst parameters

Reply:
done